crates/control/src/role_assignment.rs

```diff
@@ -109,10 +109,10 @@ impl RoleAssignment {
         }
 
         let send_game_controller_return_message = self
-            .last_transmitted_game_controller_return_message
+            .last_system_time_transmitted_game_controller_return_message
```

Comment:
In general there is no benefit of encoding type information into variables names (system time). We should configure our IDEs correctly.

```suggestion
            .last_transmitted_game_controller_return_message
```

Reply:
This is not meant as type but more changing the meaning what is in the variable as this does not contain the message itself but the timestamp of it. And even without having to use an IDE it is benefitial to have this information, IMHO